README.md

```diff
@@ -82,6 +82,15 @@ npm publish --access=public
 
 NPM will automatically build the library first, as `package.json` defines `script/prepare`.
 
+#### Production Release
+To deploy a production release:
+
+1. Merge all changes to `master`
+2. Use https://github.com/concord-consortium/portal-report/releases to create a new release tag
+3. Login to https://learn.staging.concord.org/ as an admin and under Admin > Site Admin Links > External Reports update the URL field of any External Reports that use the latest version of portal-report. Change the URL to the newly created release (e.g., https://portal-report.concord.org/version/vx.y.z/).
+4. Run any QA and testing on staging release.
+5. Once testing is complete, login to https://learn.concord.org/ as an admin and under Admin > Site Admin Links > External Reports update the URL field of any External Reports that use the latest version of portal-report. Change the URL to the newly created release (e.g., https://portal-report.concord.org/version/vx.y.z/).
+
```

Comment:
@mklewandowski , We probably should add a Step 6 to test in production once the URL in learn.concord.org is updated.

Reply:
Why bother? If you've made it this far I'm _sure_ it's fine. 😉

Reply:
Sounds good!  Step added.

Reply:
Anyone is welcome to approve this so we can merge.

Reply:
I'd be happy to approve but it looks like only @eireland can. I can only add comments to this PR as far as I can tell.

Reply:
@tejal-shah I added you as a reviewer.